html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <header>
+    <h1>Coding Form!</h1>
+  </header>
+  <form>
+    <div>
+      Email
+      <br>
+      <input type="text" name="Email"/>
+      <br>
+      Why is coding cool?
+      <br>
+      <select name="ycool">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>Favourite Language (choose one)</p>
+    <div>
+      <input type="radio" name="favlanguage" value="html"/> HTML
```

Comment:
each of these inputs needs a label, you can wrap then inputs with a label if desired